### ISSUE TITLE
Invalid references in twisted.logger

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8514]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8514|
|Type|enhancement|
|Created|2016-06-27 17:07:22Z|

Following are the invalid references in twisted.conch obtained by checking out the trunk and running

```
./bin/admin/build-apidocs . apidocs

```

```
twisted.logger._flatten.KeyFlattener: invalid ref to Formatter.parse

```

<details><summary>Searchable metadata</summary>

```
trac-id__8514 8514
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1467047242690981 1467047242690981
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See : https://github.com/twisted/twisted/pull/237

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset e60f218dfbeffabb49dfd94a507166168231df54
```
#!CommitTicketReference repository="" revision="e60f218dfbeffabb49dfd94a507166168231df54"
Merge 8514-acabhishek942-doc: Fixed invalid epydoc reference in twisted.logger

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8514
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
